### ISSUE TITLE
58410B2B - Guardian Heroes

### ISSUE THREAD

**Issue by alpargataman** [xenia-project/game-compatibility#422]
[Marketplace](http://marketplace.xbox.com/en-US/Product/GUARDIAN-HEROES-TM/66acd000-77fe-1000-9115-d80258410b2b)
Tested on (xenia ea7bad10)

# Issues:
gpu-missing-drawing, state-gameplay

# Log:
[Log file](http://www67.zippyshare.com/v/fq46he38/file.html)

# Screenshot(s):
![](http://i.imgur.com/OVumhFX.jpg)
![](http://i.imgur.com/b34sBLU.png)
![](http://i.imgur.com/VCYXoXs.png)
![](http://i.imgur.com/7R7whMd.png)

# Labels:
gpu-missing-drawing, state-gameplay

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299)

Characters are visible. (gpu-missing-drawing) can be removed.

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2146057/xenia.zip)

![untitled](https://user-images.githubusercontent.com/40533288/42046661-08c292c2-7b07-11e8-8d1d-a81476546a0d.jpg)

**Comment by Kla890** [comment]
Game work perfect, only problem is a 3 second lag audio, there is a fix?

**Comment by bdeimos** [comment]
If audio problem is not related to your system next versions of Xenia may fix it.
The best thing is to test game again in a couple months.

**Comment by admiralzeech** [comment]
Tested on vulkan build downloaded on 16/1/2019.  Played through the first level, everything fine except some channels of the audio are delayed for a few seconds.  (usually the big impact sounds?)

**Comment by bbfe** [comment]
Is it just me or is the game stuck in demo mode?

**Comment by admiralzeech** [comment]
> 
> 
> Is it just me or is the game stuck in demo mode?

On mine it does say, "Unlock full game!" in the main menu.  I didn't play through enough to find out whether it stops early or not.  Does anyone know where the demo ends?

**Comment by admiralzeech** [comment]
As of the 24th Aug 2019 build, the sound delay issue is still present.

The solution to the demo mode thing was found in the FAQ, set this in the setting file:
license_mask = -1

**Comment by Aryudsoh** [comment]
A few times in the game (first happened after about 4 stages) there were instances of sprite tearing of the playable character sprite. Looked like a square shape with a mishmash of pixels inside. Noticed it happening a few times when grabbing an enemy. Also saw some sprite tearing when fighting the old man boss during one of his special attacks. In that instance it was a black square around his character sprite.

**Comment by ZerotakerZX** [comment]
Works great on latest xenia, no slowdowns or sound delay. But during one of the games ending, when in sky sanctuary undead hero clashes swords and screen goes all white... xenia drops me to the desktop. Alas. Fix this issue and this game will be perfect.
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5936765/xenia.zip)

**Comment by belegdol** [comment]
Does not seem to work with today's master(360e2f541): sound will cut out after several seconds and everything will get slow after that.

**Comment by iBajwa93** [comment]
> Does not seem to work with today's master(360e2f541): sound will cut out after several seconds and everything will get slow after that.

Yeah I just checked, the background music starts cutting out. The latest version that works BEFORE this bug occurred was the May 6th Canary release. 

From then on they did something on the May 7th that caused the music to start cutting out.

**Comment by Clemy01** [comment]
Hello, same problem with the latest build (16/07) background music starts cutting out. :(

**Comment by Frawg1** [comment]
[v1.0.2784-master] Background music will cut out a few seconds into the game and then on next scene it will come back then cut out again soon after

**Comment by Jabnin** [comment]
+1 for BGM cutting out. Seems perfect otherwise.

**Comment by LazierTomato** [comment]
+1 on the latest version as of now too, playing it for a bit the emulation is ALMOST perfect except for the music consistently cutting out after anywhere from 5-20 seconds at seemingly random

**Comment by RenanIsola** [comment]
> +1 on the latest version as of now too, playing it for a bit the emulation is ALMOST perfect except for the music consistently cutting out after anywhere from 5-20 seconds at seemingly random

+ 1

**Comment by LoutrusMagnus** [comment]
Hello :) Do you know where I can find the game iso for DL ? Cheers

**Comment by Impruve** [comment]
Any fixes for the background music cutting off? I love that soundtrack hehe

**Comment by UltraSyko** [comment]
BGM still cuts out after a few seconds

**Comment by Sowa95** [comment]
Tested on: https://github.com/xenia-project/xenia/commit/f6b5424a9f7daee69ea7ea837b6d4224ea47c368

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/15010206/xenia.zip)

No changes since last report.
Audio seems fine on Canary. Played first 3 levels, BGM didn't cut out.

**Comment by Impruve** [comment]
> Tested on: [xenia-project/xenia@f6b5424](https://github.com/xenia-project/xenia/commit/f6b5424a9f7daee69ea7ea837b6d4224ea47c368)
> 
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/15010206/xenia.zip)
> 
> No changes since last report. Audio seems fine on Canary. Played first 3 levels, BGM didn't cut out.

thank you, canary does indeed work like a charm
